plugins/automation/spec/triggers/topic_tags_changed_spec.rb

```diff
@@ -328,5 +328,75 @@
         end
       expect(list.length).to eq(1)
     end
+
+    it "should only fire if tag is added if TAGS_ADDED is set" do
```

Comment:
For these specs, maybe you could with context blocks to make it easier to read?
```ruby
context "with TAGS_ADDED trigger" do
end
...
```ruby